tests/test_lane.py

```diff
@@ -3090,241 +3135,6 @@ def search(self, *args, **kwargs):
         Lane.search_target = old_lane_search_target
         WorkList.search = old_wl_search
 
-    def test_bibliographic_filter_clause(self):
```

Comment:
These tests were merged into the corresponding test in `DatabaseBackedWorkList`, since `Lane` no longer has any database code of its own.